internal/batches/ui/exec_ui.go

```diff
@@ -40,7 +40,7 @@ type ExecUI interface {
 
 	CreatingBatchSpec()
 	CreatingBatchSpecSuccess(previewURL string)
-	CreatingBatchSpecError(err error) error
+	CreatingBatchSpecError(lr int, err error) error
```

Comment:
Since here it's the raw int, maybe:

```suggestion
	CreatingBatchSpecError(maxUnlicensedCS int, err error) error
```